app/models/reservation.rb

```diff
@@ -388,7 +388,7 @@ def set_billable_minutes
   end
 
   def calculated_billable_minutes
```

Comment:
@jossim looks like this is only called in hook:
```
  before_save :set_billable_minutes
```
... I think I'd rather skip `set_billable_minutes` for reconciled order details - what do you think?
I'd want to test that out with Skip Review/Nonbillable but I think it makes more sense to keep the previous billable min than set a new one that we expect to be the same anyway.